packages/react/menu/src/Menu.tsx

```diff
@@ -280,6 +282,7 @@ const MenuSubContent = React.forwardRef((props, forwardedRef) => {
   return context.isSubmenu ? (
     <MenuContentImpl
       id={context.contentId}
+      aria-labelledby={context.triggerId}
```

Comment:
What is the `aria-labelledby` specifically for here and why is it only for sub triggers?

Reply:
It's not only for sub triggers, I add it to the root trigger too in `DropdownMenu`. There isn't a root trigger part in `Menu` which is why this might look a bit confusing.

Reply:
Yeah I'm starting to struggle a bit, it feels like we have code a bit all over now. It feels like we're tracking submenu stuff is both now too, which may be ok but at first glance seems repeated.

Reply:
We have to track `isSubmenu` stuff in both because there requirements in `DropdownMenu` **root** parts that don't apply in `DropdownMenu` **sub** parts. This logic cannot be moved to `Menu` because it applies specifically to `DropdownMenu`s only.

The `isSubmenu` boolean in `Menu` handles all the common submenu related stuff.

We initially tried to expose a `MenuTrigger` from `Menu` to DRY up a bunch of this stuff but `ContextMenu`s don't have triggers and it started to get really complicated when we tried it. We can maybe give that another go though.

Reply:
Yeah I realised that after, I think where we landed makes sense.